stats/src/main/java/io/airlift/stats/DecayCounter.java

```diff
@@ -38,9 +38,20 @@ public DecayCounter(double alpha)
 
     public DecayCounter(double alpha, Ticker ticker)
     {
+        this(0, alpha, ticker, TimeUnit.NANOSECONDS.toSeconds(ticker.read()));
```

Comment:
```suggestion
        this(0, alpha, requireNonNull(ticker, "ticker is null"), TimeUnit.NANOSECONDS.toSeconds(ticker.read()));
```

Reply:
I think that muddles the line too much. The call would fail with NPE due to `ticker.read()`, anyway.